test/services/platform-environment-requirements.ts

```diff
@@ -0,0 +1,210 @@
+		});
+
+		it("should show prompt when environment is not configured", async () => {
+			const doctorService = testInjector.resolve("doctorService");
+			doctorService.canExecuteLocalBuild = () => false;
+
+			let isPromptForChoiceCalled = false;
+			const prompter = testInjector.resolve("prompter");
+			prompter.promptForChoice = () => {
+				isPromptForChoiceCalled = true;
+				return PlatformEnvironmentRequirements.CLOUD_BUILDS_OPTION_NAME;
+			};
+
+			let isInstallExtensionCalled = false;
+			const extensibilityService = testInjector.resolve("extensibilityService");
+			extensibilityService.installExtension = () => {isInstallExtensionCalled = true; };
+			extensibilityService.getInstalledExtensions = () => { return {}; };
+
+			await assert.isRejected(platformEnvironmentRequirements.checkEnvironmentRequirements(platform));
+			assert.isTrue(isPromptForChoiceCalled);
+			assert.isTrue(isInstallExtensionCalled);
+		});
+
+		it("should return true when environment is configured", async () => {
+			const doctorService = testInjector.resolve("doctorService");
+			doctorService.canExecuteLocalBuild = () => true;
+
+			const result = await platformEnvironmentRequirements.checkEnvironmentRequirements(platform);
+			assert.isTrue(result);
+		});
+
+		describe("when setup script option is selected ", () => {
+			it("should return true when env is configured after executing setup script", async () => {
+				let index = 0;
+				const doctorService = testInjector.resolve("doctorService");
+				doctorService.canExecuteLocalBuild = () => {
+					if (index === 0) {
+						index++;
+						return false;
+					}
+
+					return true;
+				};
+				doctorService.runSetupScript = () => Promise.resolve();
+
+				const prompter = testInjector.resolve("prompter");
+				prompter.promptForChoice = () => Promise.resolve(PlatformEnvironmentRequirements.SETUP_SCRIPT_OPTION_NAME);
+
+				const result = await platformEnvironmentRequirements.checkEnvironmentRequirements(platform);
+				assert.isTrue(result);
+			});
+			it("should prompt for choice when env is not configured after executing setup script", async () => {
+				const doctorService = testInjector.resolve("doctorService");
+				doctorService.canExecuteLocalBuild = () => false;
+				doctorService.runSetupScript = () => Promise.resolve();
+
+				let index = 0;
+				let isPromptForChoiceCalled = false;
+				const prompter = testInjector.resolve("prompter");
+				prompter.promptForChoice = () => {
+					if (index === 0) {
+						index++;
+						return PlatformEnvironmentRequirements.SETUP_SCRIPT_OPTION_NAME;
+					}
+
+					isPromptForChoiceCalled = true;
+					return PlatformEnvironmentRequirements.CLOUD_BUILDS_OPTION_NAME;
+				};
+
+				let isInstallExtensionCalled = false;
+				const extensibilityService = testInjector.resolve("extensibilityService");
+				extensibilityService.installExtension = () => {isInstallExtensionCalled = true; };
+				extensibilityService.getInstalledExtensions = () => { return {}; };
```

Comment:
You can use the short syntax to return an object here:
`extensibilityService.getInstalledExtensions = () => ({});`